src/jit/codegencommon.cpp

```diff
@@ -4336,17 +4336,6 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
             }
         }
```

Comment:
I don't understand why you would need any of these changes in codegencommon.cpp

My checkin probably did break the handling of HFA doubles for ARM32.
But these changes in codegencommon.cpp don't look like the correct fix to me.